client/my-sites/importer/newsletter/content-upload/importing-pane.jsx

```diff
@@ -257,8 +257,12 @@ export class ImportingPane extends PureComponent {
 						siteId={ siteId }
 						sourceType={ sourceType }
 						sourceAuthors={ customData.sourceAuthors }
-						sourceTitle={ customData.siteTitle || this.props.translate( 'Original Site' ) }
-						targetTitle={ siteName }
+						sourceTitle={
+							sourceType === 'Substack'
+								? this.props.translate( 'Substack' )
+								: customData.siteTitle || this.props.translate( 'Original Site' )
```

Comment:
The sourceType check is currently unnecessary, since the component upstream getting the data is specifically calling it for substack (so it should always be substack). However, this just gives us better and more contextual fallbacks initially if there were ever any non substack sources piped through here.

Reply:
That makes sense. I think the plan was to eventually open it up to more newsletter platforms.